graylog2-web-interface/src/views/components/messagelist/CustomHighlighting.jsx

```diff
@@ -1,24 +1,22 @@
 // @flow strict
 import * as React from 'react';
+import { useContext } from 'react';
 import PropTypes from 'prop-types';
 
-import connect from 'stores/connect';
-
-import { HighlightingRulesStore } from 'views/stores/HighlightingRulesStore';
+import { ActionContext } from 'views/logic/ActionContext';
 import DecoratorContext from 'views/components/messagelist/decoration/DecoratorContext';
-import HighlightingRule from 'views/logic/views/formatting/highlighting/HighlightingRule';
 
 import PossiblyHighlight from './PossiblyHighlight';
 import Highlight from './Highlight';
 
 type Props = {
   children: ?React.Node,
   field: string,
-  highlightingRules: { [string]: Array<HighlightingRule> },
   value?: any,
 };
 
-const CustomHighlighting = ({ children, field: fieldName, value: fieldValue, highlightingRules }: Props) => {
+const CustomHighlighting = ({ children, field: fieldName, value: fieldValue }: Props) => {
+  const { highlightingRules = {} } = useContext(ActionContext);
```

Comment:
I had to use the `ActionContext` because `AdditionalContext` is not working with `useContext`, but I could also wrap the component in `AdditionalContext.Provider`, instead.